ga/latest/kernel/Dockerfile.ubi.openjdk21

```diff
@@ -0,0 +1,176 @@
+# (C) Copyright IBM Corporation 2023.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-21-jre-ubi9-minimal AS getRuntime
+
+USER root
+
+ARG VERBOSE=false
+
+# Install WebSphere Liberty
+ARG LIBERTY_VERSION=24.0.0.1
+ARG LIBERTY_URL
+ARG DOWNLOAD_OPTIONS=""
+
+# If there is a local copy of the image use that instead
+COPY resources/ /tmp/
+
+RUN microdnf -y install shadow-utils unzip wget findutils openssl \
+    && mkdir -p /licenses \
+    && useradd -u 1001 -r -g 0 -s /usr/sbin/nologin default \
+    && LIBERTY_URL=${LIBERTY_URL:-$(wget -q -O - https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/index.yml | grep -E "^\s*kernel:.*${LIBERTY_VERSION}\.zip" | sed -n 's/\s*kernel:\s//p' | tr -d '\r' )}  \
+    && if [ ! -f /tmp/wlp.zip ]; then wget $DOWNLOAD_OPTIONS $LIBERTY_URL -U UA-IBM-WebSphere-Liberty-Docker -O /tmp/wlp.zip; fi \
+    && chmod -R g+x /usr/bin \
+    && unzip -q /tmp/wlp.zip -d /opt/ibm \
+    && chown -R 1001:0 /opt/ibm/wlp \
+    && chmod -R g+rw /opt/ibm/wlp \
+    && cp -a /opt/ibm/wlp/lafiles/. /licenses/
+
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-21-jre-ubi9-minimal
+
+USER root
+
+ARG VERBOSE=false
+ARG OPENJ9_SCC=true
+
+ARG LIBERTY_VERSION=24.0.0.1
+ARG LIBERTY_BUILD_LABEL={replace_with_gm_driver_label}
+
+LABEL org.opencontainers.image.authors="Leo Christy Jesuraj, Thomas Watson, Wendy Raschke, Michal Broz" \
+      org.opencontainers.image.vendor="IBM" \
+      org.opencontainers.image.url="https://github.com/WASdev/ci.docker" \
+      org.opencontainers.image.documentation="https://www.ibm.com/support/knowledgecenter/SSAW57_liberty/com.ibm.websphere.wlp.nd.multiplatform.doc/ae/cwlp_about.html" \
+      org.opencontainers.image.version="$LIBERTY_VERSION" \
+      org.opencontainers.image.revision="$LIBERTY_BUILD_LABEL" \
+      org.opencontainers.image.description="This image contains the WebSphere Liberty runtime with IBM Semeru Runtime Open Edition OpenJDK with OpenJ9 and Red Hat's UBI 9 as the base OS.  For more information on this image please see https://ibm.biz/wl-app-image-template" \
```

Comment:
UBI 9 --> UBI 9 minimal 

Same comment for the summary and description labels below

Reply:
done